src/main/resources/application-dev.yml

```
@@ -0,0 +1,42 @@
+        format_sql: true
+        default_batch_fetch_size: 100
+
+# 로깅 설정
+logging:
+  level:
+    com.codeit.mission.deokhugam: debug
+    org.hibernate.SQL: debug
+    org.hibernate.orm.jdbc.bind: trace
+
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (3 > 2)

(empty-lines)